dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumMonitor.kt

```diff
@@ -97,7 +97,12 @@ interface RumMonitor {
      * @param attributes additional custom attributes to attach to the action
      * @see [addUserAction]
      * @see [startUserAction]
+     * @deprecated Use [stopUserAction] with name parameter instead.
      */
+    @Deprecated(
```

Comment:
Why are you deprecating this one ?

Reply:
As mentioned in the PR description:

```
This change also deprecates variant of stopUserAction method which doesn't have name parameter in order to:

be consistent with other methods of the interface
prevent users from calling it to allow better internal API changes in future
```

Do you think this change should be rejected?